modules/knock_down/action.sqf

```diff
@@ -0,0 +1,52 @@
+// AUTHOR: StatusRed (EM-Creations.co.uk)
+
+private _validWeapons = (_this select 0);
+private _reviveAction = (_this select 1);
+
+private ["_nearestunits", "_nearestunitofside", "_prevAnim"];
```

Comment:
private the variables when you define them with a datatype

Reply:
What do you mean by this?

Reply:
`private _var = value` instead of initializing them in `private [_vars]`

Reply:
Got it, it's more readable too.